packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/controller.js

```diff
@@ -0,0 +1,96 @@
+import {
+  PROOF_TYPE,
+  PAGE_TYPE,
+  MAX_SIZE,
+  KYC_ALLOWED_FILE_EXTENSIONS,
+} from '../../user-identity-documents.constant';
+
+export default class AccountUserIdentityDocumentsUploadDetailController {
+  /* @ngInject */
+  constructor() {
+    this.PROOF_TYPE = PROOF_TYPE;
+    this.PAGE_TYPE = PAGE_TYPE;
+    this.MAX_SIZE = MAX_SIZE;
+    this.multipleFile = false;
+    this.isValid = false;
+    this.badFileExtensionsFormatedList = undefined;
```

Comment:
**nitpick:** Use `null` instead. If it needs to be `undefined` then this line has no impact, so can be removed.